### PR TITLE
Feature 9162 rebased

### PR DESCRIPTION
**Pull request by CarloBar1**
Rebased branch for [previous pull request](https://github.com/Neovici/cosmoz-omnitable/pull/560), as there were too many wrong commit messages

### CODE REVIEW

File: cosmoz-omnitable-styles.js [hunk 1]
```diff
@@ -132,7 +132,28 @@ export default css`
...
+	cosmoz-omnitable-number-range-input,
+	cosmoz-omnitable-date-range-input,
+	cosmoz-omnitable-time-range-input,
+	cosmoz-omnitable-datetime-range-input,
+	paper-input {
+		--paper-font-subhead_-_font-family: var(
+			--cosmoz-header-font-family,
+			'Inter',
+			sans-serif
+		);
```
Comment (cristinecula) [inline cosmoz-omnitable-styles.js:145]: Are you sure that this works correctly? AFAIK `var` only takes 2 parameters, and you have 3.

File: cosmoz-omnitable-styles.js [hunk 2]
```diff
@@ -132,7 +132,28 @@ export default css`
...
+		text-transform: var(--cosmoz-header-text-transform, uppercase);
```
Comment (cristinecula) [inline cosmoz-omnitable-styles.js:146]: You should include the component name in the css var, so it is clear when someone is reading the CSS the purpose of a certain variable.

```suggestion
		text-transform: var(--cosmoz-omnitable-header-text-transform, uppercase);
```

**Review by cristinecula — CHANGES_REQUESTED** [review]
Almost there.

File: cosmoz-omnitable-styles.js (hunk 1 above)
Comment (CarloBar1) [reply cosmoz-omnitable-styles.js:145]: In the [CSS var() definition](https://developer.mozilla.org/en-US/docs/Web/CSS/var), it says that a var declaration can allow more than one fallback value after the first comma. It is though a bit unclear whether the fallback values are considered one after each other, so I just deleted those commas.

Comment (cristinecula) [reply cosmoz-omnitable-styles.js]: Please figure out how it is supposed to be written and update accordingly. Experiment with both variants and figure out which is doing the right thing.

File: cosmoz-omnitable-styles.js [hunk 3]
```diff
@@ -132,7 +132,33 @@ export default css`
 			--paper-font-caption_-_line-height: 18px;
 		}
 	}
+	cosmoz-omnitable-amount-range-input,
+	cosmoz-omnitable-number-range-input,
+	cosmoz-omnitable-date-range-input,
+	cosmoz-omnitable-time-range-input,
+	cosmoz-omnitable-datetime-range-input,
+	paper-input {
```
Comment (cristinecula) [inline cosmoz-omnitable-styles.js:140]: It should work the same if you defined these styles for the header containing element.
```suggestion
	[selector-for-header-container] {
```

Please try it out.

File: lib/settings/cosmoz-omnitable-settings.js [hunk 4]
```diff
@@ -144,8 +144,9 @@ const placement = ['bottom-right', ...defaultPlacement],
 				html`<div class="headerDots" slot="button">...</div>` :
 				html`
 					<svg viewBox="0 0 24 24" width="24" slot="button" fill="currentColor">
-						<path d="M10 18h5V5h-5v13zm-6 0h5V5H4v13zM16 5v13h5V5h-5z"></path>
-					</svg>`
+					<circle cx="10" cy="18" r="2"></circle>
+						<circle cx="10" cy="12" r="2"></circle>
+						<circle cx="10" cy="6" r="2"></circle></svg>`
```
Comment (cristinecula) [inline lib/settings/cosmoz-omnitable-settings.js:149]: Please post a screenshot.

**Review by cristinecula — CHANGES_REQUESTED** [review]

File: lib/settings/cosmoz-omnitable-settings.js (hunk 4 above)
Comment (CarloBar1) [reply lib/settings/cosmoz-omnitable-settings.js:149]: image

Comment (CarloBar1) [reply lib/settings/cosmoz-omnitable-settings.js:149]: Right now it looks like this:
image

File: cosmoz-omnitable-styles.js (hunk 3 above)
Comment (CarloBar1) [reply cosmoz-omnitable-styles.js]: Included all in the .header-cell selector, and it works.

**Pull request merged**